### PR TITLE
QTBSI-1119 & QTBSI-1085: Add new institutions vol. 44 and vol. 45

### PR DESCRIPTION
**Pull request by lassesuom**

### CODE REVIEW

File: world_universities_and_domains.json [hunk 1]
```diff
@@ -131197,5 +131190,389 @@
...
+  },
+  {
+    "web_pages": [
+      "https://www.ucl.br/"
+    ],
+    "name": "UCL - Faculdade do Centro Leste",
+    "alpha_two_code": "BR",
+    "state-province": null,
+    "domains": [
+      "https://www.ucl.br/"
```
Comment (t4ngth00) [inline world_universities_and_domains.json:131574]: The domain is not correct here

File: world_universities_and_domains.json [hunk 2]
```diff
@@ -131197,5 +131190,389 @@
...
+  },
+  {
+    "web_pages": [
+      "https://www.juventus.ch"
+    ],
+    "name": "Juventus Technikerschule",
+    "alpha_two_code": "CH",
+    "state-province": null,
+    "domains": [
+      "juventus.schule"
```
Comment (t4ngth00) [inline world_universities_and_domains.json:131526]: The domain here is quite strange, let's double-check with Emilia

File: world_universities_and_domains.json [hunk 3]
```diff
@@ -131197,5 +131190,389 @@
...
+    "domains": [
+      "abes.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://mitsgwalior.in/"
+    ],
+    "name": "Madhav Institute Of Science and Technology",
```
Comment (t4ngth00) [inline world_universities_and_domains.json:131234]: The correct on should be `Madhav Institute of Science and Technology`

File: world_universities_and_domains.json (hunk 2 above)
Comment (lassesuom) [reply world_universities_and_domains.json:131526]: image
Looks to be valid

File: world_universities_and_domains.json (hunk 1 above)
Comment (lassesuom) [reply world_universities_and_domains.json:131574]: Oh that is certainly wrong, thanks

**Review by Qt-nhannguyen — APPROVED** [review]
Let's correct the errors and this can be merged 👍

**Pull request merged**